### PR TITLE
Added Code Tours

### PR DESCRIPTION
**Pull request by adityanaag3**
### What does this PR do?
Adds Code Tours to the project

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### CODE REVIEW

**Comment by adityanaag3** [comment]
Hey @pchittum , I've created a few Code Tours. Can you please check and let me know if you have any suggestions?

File: .tours/introducing-lwc-recipes.tour [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "title": "👋 Introducing LWC Recipes",
```
Comment (pchittum) [inline .tours/introducing-lwc-recipes.tour:2]: probably we want to make sure this states clearly "LWC Recipes OSS"

File: .tours/introducing-lwc-recipes.tour [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "steps": [
+    {
+      "directory": "src",
+      "description": "Recipes is a collection of easy-to-digest code examples for Lightning Web Components Open Source."
```
Comment (pchittum) [inline .tours/introducing-lwc-recipes.tour:6]: "The recipes app" instead of just "Recipes" maybe?

File: .tours/introducing-lwc-recipes.tour [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    {
+      "directory": "src/modules/recipe",
+      "description": "Each recipe demonstrates how to code a specific task in about 30 lines of code or less. From Hello World to data access and third-party libraries, there is a recipe for that!"
```
Comment (pchittum) [inline .tours/introducing-lwc-recipes.tour:10]: Let's not explicitly state a number of lines. Preferably, "in the minimal lines possible".

File: .tours/introducing-lwc-recipes.tour (hunk 1 above)
Comment (pchittum) [reply .tours/introducing-lwc-recipes.tour:2]: Also, this name is too similar to "Intro to LWC". I know one is intro to the project, and the other is supposed to be intro to the framework. But I'd suggest a different title for one. Like this one could be called "Welcome to LWC OSS Recipes" or something like that. 

Like the :wave: by the way.

**Review by pchittum — COMMENTED** [review]
@adityanaag3 Just a few minor tweaks and this should be ready to go.

File: .tours/introducing-lwc-recipes.tour (hunk 1 above)
Comment (adityanaag3) [reply .tours/introducing-lwc-recipes.tour]: Done

File: .tours/introducing-lwc-recipes.tour (hunk 2 above)
Comment (adityanaag3) [reply .tours/introducing-lwc-recipes.tour]: Done

File: .tours/introducing-lwc-recipes.tour (hunk 3 above)
Comment (adityanaag3) [reply .tours/introducing-lwc-recipes.tour]: Done

**Review by pchittum — APPROVED** [review]

**Pull request merged**